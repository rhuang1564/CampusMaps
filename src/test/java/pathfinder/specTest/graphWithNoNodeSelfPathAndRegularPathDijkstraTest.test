# Find a path to nonexistant node and nonexistant self path
# with Dijkstra's algorithm
# in a no node and no edge graph

CreateGraph graph

FindPath graph A A

FindPath graph A B
