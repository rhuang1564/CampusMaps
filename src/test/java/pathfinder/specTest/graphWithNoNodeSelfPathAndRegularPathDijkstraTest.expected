# Find a path to nonexistant node and nonexistant self path
# with Dijkstra's algorithm
# in a no node and no edge graph

created graph graph

unknown node A
unknown node A

unknown node A
unknown node B